lib/prefix.js

```diff
@@ -2,11 +2,8 @@
 "use strict";
 
 var url = require("url"),
-    
```

Comment:
Leave this linebreak in, please. Separating core modules from node_modules from my vars.